# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../babytrans/babytrans-0.9.2-gcc.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN babytrans-0.9.2-orig/src/main.cc babytrans-0.9.2/src/main.cc
--- babytrans-0.9.2-orig/src/main.cc	2002-11-16 10:48:11.000000000 +0100
+++ babytrans-0.9.2/src/main.cc	2010-08-26 12:41:01.216006874 +0200
@@ -27,6 +27,7 @@
 # error "HAVE_CONFIG_H not set"
 #endif
 
+#include <assert.h>
 #include <iostream>
 #include <string>
 #include <vector>
